toolkits/springernature/packages/springernature-forms/scss/50-components/_errors.scss

```diff
@@ -0,0 +1,31 @@
+
+.c-error-summary {
```

Comment:
class name format should follow the component name, so in this case rather than `c-error-summary` it should be `c-forms__error-summary` then follow that format